Mathlib/Data/Rel.lean

```diff
@@ -262,8 +355,42 @@ namespace Function
 def graph (f : α → β) : Rel α β := fun x y => f x = y
 #align function.graph Function.graph
 
+theorem graph_id : graph id = @Eq α := by simp[graph]
+
+theorem graph_comp {f : β → γ} {g : α → β} : graph (f ∘ g) = Rel.comp (graph g) (graph f) := by
+  ext x y
+  simp[graph, Rel.comp]
+
 end Function
 
+theorem Equiv.graph_inv (f : α ≃ β) : f.invFun.graph = Rel.inv (f.toFun.graph) := by
+  ext x y
+  simp[Function.graph, Rel.inv, Equiv.apply_eq_iff_eq_symm_apply (f := f.symm), flip]
+  exact eq_comm
```

Comment:
We can automate this proof! If we remind `aesop` of the definition of `Rel.inv`:
```suggestion
  aesop (add norm Rel.inv_def)
```

Reply:
is this really the preferred way? seems more computation-heavy to me

Reply:
I don't think that 600 heartbeats is a big issue, since the boundary lies at 200 000. In general I wouldn't worry about efficiency unless you start to notice slowdowns, time for humans to read it is generally worth more than time for computers to check it.

Actually I am surprised that it's just 600 heartbeats! That's nothing :)